src/Headings/HeadingOne.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+const Heading = styled.h1`
+  font-family: ${({ theme }): string => theme.font.heading.family};
+  font-size: ${({ theme }): string => theme.font.heading.size};
+  font-weight: ${({ theme }): string => theme.font.heading.weight};
+  color: ${({ theme }): string => theme.font.heading.color};
+`;
+
```

Comment:
Missing documentation for the component itself and its props

Reply:
I added documentation for props in 76e66fd. There is an example in the `examples` folder like the other components.

Reply:
Similar to the note on the props, we should also be providing class-level doc blocks that can be picked up by `tsserver`. The examples are great for reference, but I think we need to include that API-style documentation wherever possible. So I think there is still work to be done here adding a general description for the component in a block above the declaration.

Reply:
That makes sense, though I don't see them being used elsewhere in `mark-one`...

Reply:
- [`src/Theme/MarkOneWrapper.tsx`](https://github.com/seas-computing/mark-one/blob/develop/src/Theme/MarkOneWrapper.tsx)
- [`src/Layout/Header.tsx`](https://github.com/seas-computing/mark-one/blob/develop/src/Layout/Header.tsx)
- [`src/Layout/Logo.tsx`](https://github.com/seas-computing/mark-one/blob/develop/src/Layout/Logo.tsx)

My feeling is that our documentation in the code should be as thorough and complete as possible. The styleguide is nice for actually being able to see how things will be rendered, but it's not a complete substitute for documentation.

Reply:
Ah, I hadn't seen those. I was just looking at the other components such as [BorderlessButton](https://github.com/seas-computing/mark-one/blob/develop/src/Buttons/BorderlessButton.tsx) etc.

Should I add a ticket to get documentation into the other components?

Reply:
Sure. We also need to make sure we're looking for stuff like this in code review.

Reply:
Yeah. I updated that in 8aad5a9